python/basix/ufl.py

```diff
@@ -1634,6 +1641,28 @@ def _compute_signature(element: _basix.finite_element.FiniteElement) -> str:
     return signature
 
 
+def _repr_optional_args(partial_repr: str, *args: tuple[str, _typing.Any]) -> str:
+    """Augment an element `repr` by appending non-None optional arguments.
+
+    Args:
+        partial_repr: The initial `repr` of a finite element incorporating all required
+                      arguments but not including any optional args
```

Comment:
Indent 4 spaces past the 'p' in 'partial_repr'.

Reply:
Incorporated.